css/index.css

```diff
@@ -143,4 +143,42 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+/* my start */
+html {width: 800px; margin: auto;}
+header img {display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 800px; margin-top: 20px 2%;
+}
+header {margin-bottom: 2%;}
+.navBar {width: 800px;
+    display: flex;
+    align-items: baseline;
+    padding-top: 20px;
+    margin-bottom: 20px;
+    font-size: 14px; margin-left: auto;
+    margin-right: auto;
+}
+.navBar a {justify-content: space-between;text-decoration: none; color: black; width: 20%;}
+.navBar img {width: 20%; margin-right: 5%;
+}
+.boxes > .box:first-child{background: teal;}
+.boxes > .box:nth-child(2) {background: gold}
+.boxes > .box:nth-child(3) {background: cadetblue;}
+.boxes > .box:nth-child(4) {background: coral;}
+.boxes > .box:nth-child(5) {background: crimson;}
+.boxes > .box:nth-child(6) {background: forestgreen;}
+.boxes > .box:nth-child(7) {background: darkorchid;}
+.boxes > .box:nth-child(8) {background: hotpink;}
+.boxes > .box:nth-child(9) {background: indigo;}
+.boxes > .box:last-child {background: dodgerblue;}
+/* about page */
```

Comment:
From here down, this is hard to read.  HTML or CSS doesn't care about proper indentation, but we as humans do.  There are people who like to use the least amount of lines of code as possible and I get that, but sometimes it is better for the person behind you to easily read it.  The only time I do it like this is when I have a single property I am changing like `.box1 { color: teal; }`

Reply:
Thanks usually I use codepen to clean it up but I got in a hurry there shouldn't rush things I know but I did my sprint last min and that's my fault.